bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -44,12 +44,18 @@ struct ReplicaConfig {
   // 1 <= concurrencyLevel <= 30
   uint16_t concurrencyLevel = 0;
 
-  // autoViewChangeEnabled=true , if the automatic view change protocol is enabled
-  bool autoViewChangeEnabled = false;
+  // viewChangeProtocolEnabled=true , if the view change protocol is enabled at all
+  bool viewChangeProtocolEnabled = false;
 
   // a time interval in milliseconds. represents the timeout used by the  view change protocol (TODO: add more details)
   uint16_t viewChangeTimerMillisec = 0;
 
+  // autoViewChangeEnabled=true , if the automatic view change is enabled
+  bool autoViewChangeEnabled = false;
```

Comment:
The name of this flag and feature doesn't really indicate what its doing. We can keep the name the same and add a comment, but it may be better to name it something else that shows that it is periodically rotating the leader. Maybe something like `PeriodicLeaderRotationEnabled` or  `AutoLeaderRotationEnabled`.